src/App.vue

```diff
@@ -633,14 +653,15 @@ export default {
       // set addresses
       this.storeAddresses({ data: incorporationApplication.offices || [] })
 
-      // set directors
-      // (ie, parties that have a role of Director)
-      const directors = incorporationApplication.parties?.filter(party =>
-        party.roles.filter(role =>
-          role.roleType === Roles.DIRECTOR
-        ).length !== 0
+      // Format party roles
+      incorporationApplication.parties.forEach(
+        party => party.roles?.forEach(
+          (role, index) => { party.roles[index] = role.roleType?.toLowerCase() }
```

Comment:
This is required, as the `roleType` value we receive from IA filings has a capitalized first letter, ie `Completing Party` but when requesting the role type directly from the parties, we receive all lower case. 
This is also why i refactored our ROLES enum, to be in alignment with the back end.
The parsing of filing json is much more the exception than fetching Directors or Custodians information in the Filings UI imo.

To elaborate, this functionality called here is a much rarer occurrence, in my experience it is only called after a user completes an IA and returns the filings UI with that TEMP REG, and this allows us to populate the DASHBOARD with that filing json.